detectable/src/main/java/com/synopsys/integration/detectable/detectables/conda/parser/CondaDependencyCreator.java

```diff
@@ -0,0 +1,46 @@
+/*
+ * detectable
+ *
+ * Copyright (c) 2021 Synopsys, Inc.
+ *
+ * Use subject to the terms and conditions of the Synopsys End User Software License and Maintenance Agreement. All rights reserved worldwide.
+ */
+package com.synopsys.integration.detectable.detectables.conda.parser;
+
+import com.synopsys.integration.bdio.model.Forge;
+import com.synopsys.integration.bdio.model.dependency.Dependency;
+import com.synopsys.integration.bdio.model.externalid.ExternalId;
+import com.synopsys.integration.bdio.model.externalid.ExternalIdFactory;
+import com.synopsys.integration.detectable.detectables.conda.model.CondaListElement;
+
+public class CondaDependencyCreator {
+    private static final String PYPI_CHANNEL = "pypi";
+
+    private final ExternalIdFactory externalIdFactory;
+
+    public CondaDependencyCreator(ExternalIdFactory externalIdFactory) {
+        this.externalIdFactory = externalIdFactory;
+    }
+
+    public Dependency createFromCondaListElement(CondaListElement element, String platform) {
+        if (element.channel.equals(PYPI_CHANNEL)) {
```

Comment:
```suggestion
        if (PYPI_CHANNEL.equals(element.channel)) {
```
In case element.channel is null